recipe/install_nvcc.sh

```diff
@@ -102,7 +102,14 @@ then
   LIBCUDA_SO_CONDA_NVCC_BACKUP="\${CONDA_BUILD_SYSROOT}/lib/libcuda.so-conda-nvcc-backup"
   mv "\${CONDA_BUILD_SYSROOT}/lib/libcuda.so" "\${LIBCUDA_SO_CONDA_NVCC_BACKUP}"
 fi
-ln -s "\${CUDA_HOME}/lib64/stubs/libcuda.so" "\${CONDA_BUILD_SYSROOT}/lib/libcuda.so"
+if [[ "\${CONDA_BUILD}" == 1 ]]
+then
+  echo "BUILDING"
+  ln -s "\${CUDA_HOME}/lib64/stubs/libcuda.so" "\${CONDA_BUILD_SYSROOT}/lib/libcuda.so"
+else
+  echo "NON-BUILD"
+  ln -s "\${CUDA_HOME}/lib64/stubs/libcuda.so" "\${CONDA_PREFIX}/lib/libcuda.so"
```

Comment:
Would this work?

```suggestion
  ln -s "\${CUDA_HOME}/lib64/stubs/libcuda.so" "\${CONDA_PREFIX}/lib/stubs/libcuda.so"
```

Reply:
When I activate the env, I get this error:
```
pgali@dt07:/nvme/0/pgali/cudf$ conda activate /nvme/0/pgali/envs/cudfdev
ln: failed to create symbolic link '/nvme/0/pgali/envs/cudfdev/lib/stubs/libcuda.so': No such file or directory
```

Reply:
```suggestion
  mkdir -p "\${CONDA_PREFIX}/lib/stubs"
  ln -s "\${CUDA_HOME}/lib64/stubs/libcuda.so" "\${CONDA_PREFIX}/lib/stubs/libcuda.so"
```

Reply:
Thanks @jakirkham ! This works, was able to build and verify imports/tests.